.circleci/config.yml

```diff
@@ -3,37 +3,37 @@ version: 2.1
 default_steps: &default_steps
   steps:
     - checkout
-    # Install PhantomJS (will soon go way when we stop supporting it)
-    - run: >-
-          sudo apt-get update && sudo apt-get install -y fontconfig
-          && wget -O /tmp/phantomjs.tar.bz2 https://bitbucket.org/ariya/phantomjs/downloads/phantomjs-2.1.1-linux-x86_64.tar.bz2
-          && tar -xjf /tmp/phantomjs.tar.bz2 -C /tmp
-          && sudo mv /tmp/phantomjs-2.1.1-linux-x86_64/bin/phantomjs /usr/local/bin/phantomjs
     - run: gem update --system
     # We need this specific version of bundler for our Gemfile. (v2 doesn't work)
     - run: gem install bundler --version '1.7'
+    - run: npm install
     - run: bundle install
-    - run: bundle exec rspec
+    - run: bundle exec rake snapshots
```

Comment:
Removed the custom rake script. (Added it initially because in the node world we always have a `snapshots` script, no other reason.) Invoking `percy` directly instead, via `npx percy`.